app/domain/audits/filter.rb

```diff
@@ -11,6 +19,16 @@ def audit_status=(value)
                       end
     end
 
+    def allocated_policy
+      if allocated_to == 'no_one'
+        Policies::Unallocated
+      elsif allocated_to.blank?
+        Policies::AllocatedAndUnallocated
```

Comment:
One further thing that has just occurred to me ...

`Policies::Unallocated` is supposed to be the source of truth as to what an unallocated content item is. Likewise `Policies::Allocated` for allocated. So when we want both allocated and unallocated we should apply both policies.